PUBLISH.md

```diff
@@ -0,0 +1,26 @@
+# Publishing
```

Comment:
Notes for how to publish new Docker images to Bintray if we need to (Used by the Marathon guide).